.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- '6'
+- '8'
```

Comment:
Any repercussions from this?

Reply:
@jeremypress we will have to update to node 8, which is pretty widely adopted at Box. I have also been using node 8 for the past few days and it has been fine.